gcp/databases/cloudsql/postgresql/read_replica.tf

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  replicas = {
```

Comment:
> locals where added to the locals.tf file here: https://github.com/mesoform/terraform-infrastructure-modules/pull/103/files#diff-ef69d3490fb90759543360981d5efd129975e9c5869e54f9217bcd771375c89f or you mean something else?  

@danperezsan The `replicas` local variable should also be in that locals file, as `locals.tf` is normally referened as symlink in unit tests. Local variables only need to be in `locals{}` blocks outside of the `locals.tf` file if they reference resources or data sources, as those local variables would not be able to be tested in unit tests